deploy/src/main/resources/README.md

```diff
@@ -10,106 +10,168 @@ The Job Service is a RESTful Web Service and provides a simple API.  It tracks t
+      </tr>
+      <tr>
+        <td>CAF_LANG_DETECT_WORKER_OUTPUT_FOLDER</td>
+        <td>The folder in which the Language Detection worker places result files.</td>
       </tr>
       <tr>
         <td>JOB_SERVICE_DEMO_INPUT_DIR</td>
-        <td>./input&#8209;files</td>
-        <td>This directory is made available as a source for input files which may be read. The glob-pattern is passed in as a parameter but this is the base directory that it starts from; it cannot read any files which are outside this directory.</td>
+        <td>The directory where the test files are located on the host.</td>
       </tr>
       <tr>
         <td>JOB_SERVICE_DEMO_OUTPUT_DIR</td>
-        <td>./output&#8209;files</td>
-        <td>This directory is used for storing the output from the Language Detection operation.</td>
+        <td>The output directory for test results on the host.</td>
       </tr>
     </table>
 
-    In order to run multiple instances of the demonstration stack simultaneously it would be necessary to set the `JOB_SERVICE_PORT` parameter to different values for each instance.
+4. Deploy the services  
+	Deploy the persistent volume by issuing the following command from the directory where you have downloaded the files to:
 
-3. Deploy the services  
-    First navigate to the folder where you have downloaded the files to and then run one of the following commands, depending on whether you are using Docker Compose or Docker Stack:
+    `kubectl create -f jobservice-persistentvolumeclaim.yaml`
 
-    <table>
-      <tr>
-        <td><b>Docker Compose</b></td>
-        <td>docker-compose up</td>
-      </tr>
-      <tr>
-        <td><b>Docker Stack</b></td>
-        <td>docker stack deploy --compose-file=docker-compose.yml jobservicedemo</td>
-      </tr>
-    </table>
+	Deploy the database and RabbitMQ by issuing the following command from the directory where you have downloaded the files to:
+
+    ```
+	source ./kubernetes.env \
+            ; cat jobservice-deployment.yaml \
+            | perl -pe 's/\$\{(\w+)\}/(exists $ENV{$1} && length $ENV{$1} > 0 ? $ENV{$1} : "NOT_SET_$1")/eg' \
+            | kubectl create -f -
+	```
 
-4. Navigate to the Job Service UI  
+5. Navigate to the Job Service UI  
     The Job Service is a RESTful Web Service and is primarily intended for programmatic access, however it also ships with a Swagger-generated user-interface.
 
     Using a browser, navigate to the `/job-service-ui` endpoint on the Job Service:  
 
-        http://docker-host:9411/job-service-ui
+        http://<docker-host>:<JOB_SERVICE_8080_SERVICE_PORT>/job-service-ui
 
-    Adjust 'docker-host' to be the name of your own Docker Host and adjust the port if you are not using the default.
+    Adjust `docker-host` to be the name of your own Docker Host and adjust the `JOB_SERVICE_8080_SERVICE_PORT` to match what you set it to in the `kubernetes.env` file.
```

Comment:
Is this the right terminology for Kubernetes?  Do we even have access to the Docker Host?

Reply:
Not sure, I used the IP address of my minikube machine when testing. However what this value would be for other kubernetes systems I don't know.